### PR TITLE
Clan leader can delete posts from his clan thread

### PR DESCRIPTION
**Pull request by sprunk**

### CODE REVIEW

File: Zero-K.info/Controllers/ForumController.cs [hunk 1]
```diff
@@ -66,6 +66,8 @@ public ActionResult DeletePostPrompt(int? postID)
             //int index = post.ForumThread.ForumPosts.IndexOf(post);
             var page = GetPostPage(post);
 
+            if (!Global.IsModerator && !((thread.Clan == Global.Account.Clan) && Global.Account.HasClanRight(x => x.RightKickPeople))) return Content("No permission to delete this post");
```
Comment (Licho1) [inline Zero-K.info/Controllers/ForumController.cs]: use RightEditTexts ?

**Comment by Licho1** [comment]
Does the GUI actually allow clan leader to delete anything? Where is the button?

**Comment by sprunk** [comment]
Same button as regular post delete

**Comment by Licho1** [comment]
Too expensive change, you do fairly complex query for each post displayed.

**Pull request closed without merging**